src/core/Status/Status.stories.tsx

```diff
@@ -52,30 +72,36 @@ const mockParametersWithStatus = (indicator) => {
 
 export const None = {
   parameters: mockParametersWithStatus("none"),
-  render: () => <Status statusUrl={statusUrl} />,
+  render: () =>
```

Comment:
You may be able to get a lot of this down by representing stuff [within a decorator](https://storybook.js.org/docs/writing-stories/decorators)

Reply:
Sure, nothing wrong with what's here